features-json/avif.json

```diff
@@ -568,7 +568,7 @@
       "3.0-3.1":"n"
     }
   },
-  "notes":"Supported in Edge 116 and newer only on Windows 11 (Moment 3 or Later)",
+  "notes":"Supported in Edge 116 (Canary only) on Windows Insider builds.",
```

Comment:
Does it mean the following?

- You have switched to Windows Insider build (We have to know which channel you chose though)
- You have confirmed the shortcut for Edge Canary doesn't contain the `--enable-features=msEdgeAVIF`

Reply:
So far, I could confirm that with Edge Canary 116 and 117 without the `--enable-features=msEdgeAVIF`:
- Avif is supported on Windows 11 (22621.1848 Insider Release Preview Channel) 
- Avif is **NOT** supported on Windows 22H2 (19045.3324 stable channel)
- Avif is **NOT** supported on [Windows 10 22H2 (19045.3086)](https://stackoverflow.com/questions/75459594/why-doesnt-edge-support-avif-images#comment135313441_76481551) nor on my Windows 22H2 (19045.3324 stable channel)
- Avif is supported on [Windows 102H2 (19045.3208)](https://stackoverflow.com/questions/75459594/why-doesnt-edge-support-avif-images#comment135225606_76481551) which I'm waiting to confirm that is on an Insider Channel because it is between the two versions above.

With Canary 118 however, avif is also no longer working on my Windows 11 machine.
So maybe it was just activated by mistake in those builds and has been turned off again.

Anyway, looks like at the moment no up-to-date versions of edge do support avif.
So I'd just update the PR to remove the note completely.
What do you think?

Reply:
> So far, I could confirm that with Edge Canary 116 and 117 without the --enable-features=msEdgeAVIF:
> Avif is supported on Windows 11 (22621.1848 Insider Release Preview Channel)

I forgot Windows 11 is in the release preview channel. The build # is 22621.2215, which is newer than yours.
Do you forget whether you have added `--enable-features=msEdgeAVIF` to the commands of your Edges' shortcuts?

Reply:
> Do you forget whether you have added --enable-features=msEdgeAVIF to the commands of your Edges' shortcuts?

Nope, I've never used the feature flag

Reply:
Hmm, so based on [this comment](https://stackoverflow.com/questions/75459594/why-doesnt-edge-support-avif-images/76481551?noredirect=1#comment135717629_76481551), it looks like being on a Windows Insider Build was not a requirement to have AVIF working... Weird.
Could it have been some A/B Testing?

Reply:
For me it doesn't work anywhere anymore with 118, regardless of whether it is with or without flag.
So since you already added a note that AVIF works from 114 to 116 with the flag, I would suggest to entirely remove this note